e2e-tests/demand-backup-incremental/run

```diff
@@ -43,18 +43,18 @@ run_recovery_check() {
 	local base=${3:-true}
 
 	wait_restore "${backup_name}" "${cluster}" "requested" "0" "3000"
-	echo
+	log ""
```

Comment:
we can continue using `echo` for empty lines

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1836/commits/2280fca1383717aeb8d77952aa00089554335783